.ci/env/daal4py-dpcpp.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
I would change name to dpcpp.sh  as from env perspective it's not related to daal4py